### PR TITLE
💥 add ci, enable clang-format, misc. fixes

### LINKED ISSUE #1: Add testing and CI
Add some test vectors and run against different Python versions

### LINKED ISSUE #2: Run clang-format on the output
I had to remove a change from @noahp to run clang-format on the output. For some reason, the `subprocess` call was hanging. Need to dig into it more.

### PR DESCRIPTION
**Pull request by noahp**
Add gi (github actions) via Dockerfile + tox, testing on
py27,py36,py37,py38.

Enable clang-format (at least, clang-format-10 on my machine will hang
with no args waiting for stdin, so pass `--help` to skip it).

Fix some pylint/import errors and py3 compatibility errors.

Fixes #1.
Fixes #2.

### CODE REVIEW

**Comment by noahp** [comment]
You'll need to enable github actions for this repo for the test to run. Example here: https://github.com/noahp/cpputest_mockify/pull/1/checks?check_run_id=710191506

File: cpputest_mockify.py [hunk 1]
```diff
@@ -264,6 +286,13 @@ def create_mock(input_filepath, output_folder):
         for match in FunctionParser.FUNC_REGEX.finditer(input_header):
             f_out.write("\n" + str(FunctionParser(match.group(0))) + "\n")
 
+    # with clang-format available, reflow the output file too!
+    if which("clang-format --help"):
```
Comment (noahp) [inline cpputest_mockify.py:293]: Nah, this fails shutilwhich.which. Fixing.

Comment (noahp) [reply cpputest_mockify.py:293]: ok fixed.

**Comment by kesyog** [comment]
😍 thanks so much!

**Pull request merged**